sdk/servicebus/azure-servicebus/tests/test_sessions.py

```diff
@@ -128,15 +127,15 @@ def test_session_by_queue_client_conn_str_receive_handler_receiveanddelete(self,
             with sb_client.get_queue_receiver(servicebus_queue.name, 
                                               session_id=session_id, 
                                               receive_mode=ReceiveMode.ReceiveAndDelete, 
-                                              max_wait_time=5) as session:
-                for message in session:
+                                              max_wait_time=5) as receiver:
+                for message in receiver:
                     messages.append(message)
-                    assert session_id == session._session_id
+                    assert session_id == receiver._session_id
```

Comment:
Should we make this a public attribute on `receiver.session`?
e.g. `assert session_id == receiver.session.session_id` or something? (can be a future addition - not needed now)

Reply:
Oh nevermind - I see it already it :)